general/envsetup/envsetup.go

```diff
@@ -50,8 +50,8 @@ func NewEnvSetupCommand(url string) *EnvSetupCommand {
 }
 
 func (ftc *EnvSetupCommand) Run() (err error) {
-	ftc.progress.SetHeadlineMsg("To complete your JFrog environment setup, please fill out the details in your browser")
-	time.Sleep(5 * time.Second)
+	ftc.progress.SetHeadlineMsg("Just fill out its details in your browser")
```

Comment:
The new message sounds a bit out of context to me.

Reply:
The message would indeed be out of context withput this mew PR in jfrog-cli - https://github.com/jfrog/jfrog-cli/pull/1487
The above PR adds the following lines before the above message - 
```
	fmt.Println(coreutils.PrintTitle("Thank you for installing JFrog CLI! 🐸"))
	fmt.Println(coreutils.PrintTitle("We'll now set up a FREE JFrog environment in the cloud for you, and configure your local machine to use it."))
	fmt.Println("Your environment will be ready in less than a minute.")
```
So this is how it looks like together - 
![image](https://user-images.githubusercontent.com/7830056/160273216-d40df35c-b81e-4cdd-8c83-bb6c8ee53e8a.png)